h/api/search/core.py

```diff
@@ -10,10 +10,13 @@
 log = logging.getLogger(__name__)
 
 
-def search(request_params, user=None, search_normalized_uris=False):
+def search(request, request_params, user=None, search_normalized_uris=False):
```

Comment:
The thing is it shouldn't need to pass the request, it should just pass the specific thing that it needs from the request (the hashids salt or whatever), but I wanted to tidy this up in a follow-on pull request 'cos it's gonna make a lot of noise (lot of tests to tweak I think)